### PR TITLE
Auth log messages

### PR DESCRIPTION
**Pull request by deffrian**
Fixes | Closes | Resolves #

> Anything marked as optional that you didn't need to fill in, please remove it from the PR description. Choose one of the keywords above to refer to the issue this PR solves, followed by the issue number (e.g Fixes # 666). If there is no issue, remove the line. Remove this note after reading.

## Changes:
- Auth log messages have been improved

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ x ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Other (please describe): 

## Testing
**Requires testing**

- [ ] Yes
- [ x ] No

**In case you checked yes, did you write tests??**

- [ ] Yes
- [ ] No

**Comments about testing , should you have some** (optional)

## Further comments (optional)

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: src/Nethermind/Nethermind.Core/Authentication/MicrosoftJwtAuthentication.cs [hunk 1]
```diff
@@ -110,24 +144,42 @@ public bool Authenticate(string? token)
...
-                if (_logger.IsTrace) _logger.Trace($"Authentication: authenticated. Token: {token}, iat: {iat}, time: {now}");
+                if (_logger.IsTrace)
+                    _logger.Trace($"Message authenticated. Token: {token}, iat: {iat}, time: {now}");
                 return true;
             }
 
-            if (_logger.IsInfo) _logger.Info($"Authentication: incorrect 'iat': {iat}, now: {now}");
+            if (_logger.IsInfo)
+                _logger.Info(
+                    $"Authentication token is outdated. Now is {now} UNIX time, token issued at {iat}");
```
Comment (LukaszRozmej) [inline src/Nethermind/Nethermind.Core/Authentication/MicrosoftJwtAuthentication.cs:162]: Should we format linux time in user readable way?

**Review by LukaszRozmej — COMMENTED** [review]
What about Token not being configured properly or at all? Like a message to check jsonRpcConfig.JwtSecretFile is configured correctly/at all?

**Review by MarekM25 — CHANGES_REQUESTED** [review]
Think about user who knows nothing about JWT and auth.

I expect in logs: info:

- which settings user should set
- what file was used to read the secret
- if the file was generated by Nethermind or just read from file
- Technical details of auth problem (auth null, incorrect token and so on)

**Review by LukaszRozmej — APPROVED** [review]
Pushed some small refactors + disposing file streams

**Pull request merged**